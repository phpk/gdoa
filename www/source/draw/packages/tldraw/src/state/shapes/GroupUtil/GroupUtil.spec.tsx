import { Group } from '..'

describe('Group shape', () => {
  it('Creates a shape', () => {
    expect(Group.create({ id: 'group' })).toMatchSnapshot('group')
  })
})
