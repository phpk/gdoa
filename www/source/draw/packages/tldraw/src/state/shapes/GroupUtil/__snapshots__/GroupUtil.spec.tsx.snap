// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Group shape Creates a shape: group 1`] = `
Object {
  "childIndex": 1,
  "children": Array [],
  "id": "group",
  "name": "Group",
  "parentId": "page",
  "point": Array [
    0,
    0,
  ],
  "rotation": 0,
  "size": Array [
    100,
    100,
  ],
  "style": Object {
    "color": "black",
    "dash": "draw",
    "isFilled": false,
    "scale": 1,
    "size": "small",
  },
  "type": "group",
}
`;
